Sharpmake/Options.CSharp.cs

```diff
@@ -546,6 +546,16 @@ public enum SonarQubeExclude
                 Disabled,
                 Enabled
             }
+			
+            /// <summary>
+            /// Controls whether the project is published when running a publish command
```

Comment:
Seems this property is only applicable for dotnet sdk projects, could you indicate that in the comments please?

Reply:
Thanks. Done!